internal/iam/command_rbac_role_binding.go

```diff
@@ -159,10 +161,13 @@ func (c *roleBindingCommand) parseCommon(cmd *cobra.Command) (*roleBindingOption
 /*
 Helper function to add flags for all the legal scopes/clusters for the command.
 */
-func addClusterFlags(cmd *cobra.Command, isCloudLogin bool, cliCommand *pcmd.CLICommand) {
-	if isCloudLogin {
+func addClusterFlags(cmd *cobra.Command, cfg *config.Config, cliCommand *pcmd.CLICommand, dynamicContext *dynamicconfig.DynamicContext) {
+	if cfg.IsCloudLogin() {
 		cmd.Flags().String("environment", "", "Environment ID for scope of role-binding operation.")
 		cmd.Flags().Bool("current-environment", false, "Use current environment ID for scope.")
+		if cfg.IsTest || featureflags.Manager.BoolVariation("cli.flink.open_preview", dynamicContext, config.CliLaunchDarklyClient, true, false) {
```

Comment:
what's the plan of this feature flag? will it be removed after all features are released?

Reply:
Yes, this'll be able to be removed after open preview

Reply:
yea, we have an OP flink for flink that we are going to get rid as soon we release